backend-api/src/functions/services/clientCredentialsService/clientCredentialsService.test.ts

```diff
@@ -210,11 +212,11 @@ describe("Client Credentials Service", () => {
           });
 
           clientCredentialsService.resetCache();
-          await clientCredentialsService.getClientCredentials();
+          await clientCredentialsService.getStoredClientCredentials();
```

Comment:
I think this should be getRegisteredClientCredentials to align with our domain naming?